### ISSUE TITLE
{Freeway-Bug} [Adaptive cards> CalendarReminder.Json]: Screen reader is not announcing the already selected value of the 'Snooze for ' combobox.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7828]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the 'CalendarReminder.json' on the left pane through tab key and activate it.
4. Now, navigate through tab key to the 'Snooze for combobox.
5. Observe the issue.

**Actual Result:**
On 'Snooze for' combo box, screen reader is not announcing the already selected value (i.e 5 minutes). Narrator is announcing as **'Combo box collapsed'** only.

**Expected Result:**
On 'Snooze for' combo box, screen reader should properly announce the already selected value (i.e 5 minutes). Narrator should announce as **'5 minutes selected Snooze for combobox collapsed'.**

**Observation:**
Same issue repro with 'Snooze for' combobox of 'CalendarReminderwithlabel.json'.

**Note:**
Same issue is repro with 'Color' combobox.
**User Impact:**
Screen reader user will not be able to understand that whether something is already selected or not due to which user will not be able to interact with the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/201272119-917491bc-4609-496b-a67b-4b9a1b1d0f79.mp4

https://user-images.githubusercontent.com/104483005/189749851-f9cc4f12-0dca-4411-92ad-36074b78f4cb.mp4

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue is repro'ing in below test environment, hence reactivating the bug.
**Test Environment:**
OS version: 22H2(Build 25272.1000)
Application: Adaptive Cards Visualizer (3.0.2210.26001 (latest) build.)

**Comment by JeanRoca** [comment]
This issue has been fixed and will be included in our next release. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vaishali1397** [comment]
Issue has been fixed in below test Environment, hence Adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25314. 1010)
Application: Adaptive Card Visualizer (Version 3.2.2303.22003)


**PFA for Reference**: Snooze for selected value~ Fixed issue.mp4

https://user-images.githubusercontent.com/104483005/228220578-f5842860-371b-40fe-8253-c8094b690b69.mp4
